autode/opt/optimisers/prfo.py

```diff
@@ -39,6 +41,8 @@ def __init__(
         self.alpha = Distance(init_alpha, units="ang")
         assert self.alpha > 0
         self.recalc_hessian_every = int(recalc_hessian_every)
+        self._mode_idx = imag_mode_idx
```

Comment:
```suggestion
        self._mode_idx = 0 if imag_mode_idx is None else imag_mode_idx
```

then can get rid of the logic below `# not given, always take first mode but print overlap`

Reply:
@t-young31 What I tried to do is that if `imag_mode_idx` is not set, then the first mode is followed without checking the overlap. But if `image_mode_idx` is set to 0, instead of None, then after the first iteration, it checks the overlap and chooses the mode that has highest overlap with 0th mode. I am not sure if this behaviour makes sense - happy to change it to always follow mode by overlap.

Reply:
@t-young31 From reading the manual Q-Chem always follows the chosen mode by checking overlap, whereas ORCA does not check overlap.

Reply:
hmm, I would have thought mode following is always best but again not sure. what's your opinion?